extensions/podman/packages/extension/src/checks/macos-checks.ts

```diff
@@ -100,3 +100,37 @@ export class MacPodmanInstallCheck extends BaseCheck {
     });
   }
 }
+
+export class MacKrunkitPodmanMachineCreationCheck extends BaseCheck {
+  title = 'Krunkit Installation';
+  async execute(): Promise<extensionApi.CheckResult> {
+    // we need to check if brew is installed to avoid unexpected error
+    try {
+      await extensionApi.process.exec('which', ['brew']);
+    } catch (err) {
+      // brew is not installed so do not check if krunkit has been installed with brew
+      return this.createSuccessfulResult();
+    }
+
+    // brew is installed, check if libKrun has been installed with brew
+    try {
+      const result = await extensionApi.process.exec('brew', ['list', '--verbose', 'krunkit'], {
+        env: { HOMEBREW_NO_AUTO_UPDATE: '1', HOMEBREW_NO_ANALYTICS: '1' },
+      });
+      if (!result.stderr) {
+        return this.createSuccessfulResult();
+      }
+    } catch (e: unknown) {
+      console.error(`Krunkit check failed with error: ${e}`);
+    }
+
+    return this.createFailureResult({
+      description:
+        'You dont have krunkit installed, run "brew update && brew install krunkit" to install latest version',
```

Comment:
```suggestion
        'You dont have krunkit installed. Run "brew update && brew install krunkit" to install latest version',
```

For consistency with message below.